tests/src/python/test_qgsserver_wms_getmap_size_project.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""QGIS Unit tests for QgsServer MaxHeight and MaxWidth Override Options.
+
+From build dir, run: ctest -R PyQgsServerGetMapSize -V
+
+.. note:: This test needs env vars to be set before the server is
+          configured for the first time, for this
+          reason it cannot run as a test case of another server
+          test.
+
+.. note:: This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+"""
+__author__ = 'Marco Bernasocchi'
+__date__ = '01/04/2019'
+__copyright__ = 'Copyright 2019, The QGIS Project'
+# This will get replaced with a git SHA1 when you do a git archive
+__revision__ = '$Format:%H$'
+
+import os
+
+# Needed on Qt 5 so that the serialization of XML is consistent among all
+# executions
+os.environ['QT_HASH_SEED'] = '1'
+
+
+import urllib.parse
+
+from qgis.testing import unittest
+
+from test_qgsserver import QgsServerTestBase
+
+
+class TestQgsServerWMSGetMapSizeProject(QgsServerTestBase):
+    """QGIS Server WMS Tests for GetFeatureInfo request"""
+
+    # Set to True to re-generate reference files for this class
+    regenerate_reference = False
+
+    def setUp(self):
+        os.environ['QGIS_SERVER_WMS_MAX_WIDTH'] = '6000'
+        os.environ['QGIS_SERVER_WMS_MAX_HEIGHT'] = '6000'
+        super(TestQgsServerWMSGetMapSizeProject, self).setUp()
+        self.project = os.path.join(self.testdata_path, "test_project_with_size.qgs")
+        self.expected_too_big = self.strip_version_xmlns(b'<ServiceExceptionReport version="1.3.0" xmlns="http://www.opengis.net/ogc">\n <ServiceException code="InvalidParameterValue">The requested map size is too large</ServiceException>\n</ServiceExceptionReport>\n')
```

Comment:
```suggestion
        self.expected_too_big = self.strip_version_xmlns(b'<ServiceExceptionReport version="1.3.0" xmlns="http://www.opengis.net/ogc">\n <ServiceException code="Size error">The requested map size is too large</ServiceException>\n</ServiceExceptionReport>\n')
```

Reply:
Not sure about this one, the original test used InvalidParameterValue. @pblottiere?